web-components/src/components/table-advanced/src/decorators.ts

```diff
@@ -0,0 +1,149 @@
+// DEBOUNCE decorator
```

Comment:
@havarez are these decorators table specific? i mean depounce we might want in a lot of places as well. Can we bring them out of the table folder?

Reply:
@yananym debounce as function is already in proj
decorator form is only used for Table 
if somebody wants it for another comp then he/she will move it in separate folder
until then I prefer to keep code used for specific comp close to comp